package.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "@bufferapp/bufflog",
-  "version": "0.5.3",
+  "version": "0.5.5",
   "description": "logger for all javascript and typescript Buffer services",
   "main": "dist/bufflog.js",
   "scripts": {
     "start": "./node_modules/typescript/bin/tsc --pretty && node ./dist/index.js",
-    "build": "tsc",
-    "pre-publish": "tsc",
+    "build": "./node_modules/typescript/bin/tsc",
```

Comment:
Not entirely sure what this change is. Leaving a comment to say that it's been noted, but I'm unsure how it impacts the project. Seeing it under the commit titled Cleanup

Reply:
@kiriappeee, I initiall added those entries and corrected them. For the `pre-publish` it was already set at the root level, so the one I added is not necessary :)